test/acceptance/account_test.rb

```diff
@@ -22,8 +22,8 @@ def test_changing_email
       fill_in 'email', with: 'some@email.com'
       click_on 'Update'
 
-      assert_content 'Updated email address.'
-      assert_equal 'some@email.com', find('[name=email]').value
+      assert_content 'Updated account settings.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.